setup_cfg_fmt.py

```diff
@@ -192,14 +187,15 @@ def _python_requires(
         if (
                 env.startswith('py') and
                 len(env) == 4 and
-                env[2:].isdigit()
+                env[2:].isdigit() and
+                env[2] == '3'
```

Comment:
just noticed this (unrelated) -- I think this misses `py311` as an env name

Reply:
#182